Header header
bool takeoff_land_trigger 
# 1 for takeoff 0 for landing
